mrtarget/common/ElasticsearchQuery.py

```diff
@@ -270,27 +239,45 @@ def get_associations_for_disease(self, disease, fields = None, size = 100, get_t
                                   )
         return AssociationSummary(res)
 
+    def get_all_validated_evidence_strings(self,  size=1000, datasources = []):
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html
+        index_name = Loader.get_versioned_index(Config.ELASTICSEARCH_VALIDATED_DATA_INDEX_NAME+'*', True)
 
-    def get_validated_evidence_strings(self,  size=1000, datasources = []):
+        doc_type = None
+        if datasources:
+            doc_type = datasources
 
+        res = helpers.scan(client=self.handler,
+                           query={
+                               "query": {
+                                    "match_all": {},
+                               },
+                               '_source': True,
+                               'size': size,
+                           },
+                           scroll='12h',
+                           doc_type=doc_type,
+                           index=index_name,
+                           timeout="20m",
+                           )
 
+        for hit in res:
+            yield hit['_source']
 
+    def get_validated_evidence_strings(self,  size=1000, datasources = [], is_valid=True):
         # https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html
         index_name = Loader.get_versioned_index(Config.ELASTICSEARCH_VALIDATED_DATA_INDEX_NAME+'*', True)
-        doc_type = None
 
+        doc_type = None
         if datasources:
             doc_type = datasources
+
         res = helpers.scan(client=self.handler,
                            query={
                                "query": {
-                                   "match": {
-                                       "is_valid": {
-                                           "query": True,
-                                           "type": "phrase"
-                                       }
-                                   }
-
+                                    "match_phase": {
```

Comment:
@afaulconbridge be careful! typo there! is `match_phrase`
https://www.elastic.co/guide/en/elasticsearch/guide/current/phrase-matching.html